gulpfile.js

```diff
@@ -173,6 +173,28 @@ function compile(watch, shouldMinify, opt_preventRemoveAndMakeDir,
     includePolyfills: true,
   });
 
+  compileJs('./3p/', 'ampcontext.js',
```

Comment:
Well the reason that it is split into 2 files, ampcontext and ampcontext-lib, is because firing the 'windowContextCreated' event from within the AmpContext constructor sends it before the instance is actually ready. Whether we actually need 2 binaries is a good question. Are the binaries only made for files that ad networks should get access to off our cdn? In that case I guess we don't need one for ampcontext.js
@keithwrightbos